core/src/main/java/org/apache/servicecomb/core/definition/SchemaMeta.java

```diff
@@ -57,21 +57,26 @@
   private SwaggerToClassGenerator swaggerToClassGenerator;
 
   public SchemaMeta(Swagger swagger, MicroserviceMeta microserviceMeta, String schemaId) {
-    this.packageName = SchemaUtils.generatePackageName(microserviceMeta, schemaId);
+    try {
+      this.packageName = SchemaUtils.generatePackageName(microserviceMeta, schemaId);
 
-    this.swagger = swagger;
-    this.name = schemaId;
+      this.swagger = swagger;
+      this.name = schemaId;
 
-    this.microserviceMeta = microserviceMeta;
-    this.microserviceQualifiedName = microserviceMeta.getName() + "." + schemaId;
+      this.microserviceMeta = microserviceMeta;
+      this.microserviceQualifiedName = microserviceMeta.getName() + "." + schemaId;
 
-    swaggerToClassGenerator = new SwaggerToClassGenerator(microserviceMeta.getClassLoader(), swagger, packageName);
-    swaggerIntf = swaggerToClassGenerator.convert();
+      swaggerToClassGenerator = new SwaggerToClassGenerator(microserviceMeta.getClassLoader(), swagger, packageName);
+      swaggerIntf = swaggerToClassGenerator.convert();
 
-    createOperationMgr("schemaMeta " + schemaId + " operation mgr");
-    operationMgr.setRegisterErrorFmt("Operation name repeat, schema=%s, operation=%s");
+      createOperationMgr("schemaMeta " + schemaId + " operation mgr");
+      operationMgr.setRegisterErrorFmt("Operation name repeat, schema=%s, operation=%s");
 
-    initOperations();
+      initOperations();
+    } catch (Throwable e) {
```

Comment:
It's better to log the error with the exception, so we can get the stack trace of it.

Reply:
This will be logger in main thread or by user code. Here do not need, or will print twice